packages/@react-aria/selection/src/useSelectableList.ts

```diff
@@ -107,7 +107,7 @@ export function useSelectableList(props: SelectableListOptions): SelectableListA
   // If not virtualized, scroll the focused element into view when the focusedKey changes.
   // When virtualized, Virtualizer handles this internally.
   useEffect(() => {
-    if (!isVirtualized && selectionManager.focusedKey) {
+    if (!isVirtualized && selectionManager.focusedKey && ref.current) {
```

Comment:
Perhaps
```suggestion
    if (!isVirtualized && selectionManager.focusedKey && ref?.current) {
```
since `ref` could technically be undefined? Or we could move to `let element = ref?.current?.querySelector...`

Reply:
I prefer the optional chaining op.

In this case though I think you know better in how many cases this hook is utilized.

I use this hook indirectly via [`useListBox`](https://react-spectrum.adobe.com/react-aria/useListBox.html), there the `ref` is required.
https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/listbox/src/useListBox.ts#L52